elliottlib/cli/attach_cve_flaws_cli.py

```diff
@@ -29,7 +33,8 @@
               help='Run for all advisories values defined in [group|releases].yml')
 @click.pass_obj
 @click_coroutine
-async def attach_cve_flaws_cli(runtime: Runtime, advisory_id: int, noop: bool, default_advisory_type: str, into_default_advisories: bool):
+async def attach_cve_flaws_cli(runtime: Runtime, advisory_id: int, id: str, noop: bool,
```

Comment:
Type of the multi option is `Tuple`:
```suggestion
async def attach_cve_flaws_cli(runtime: Runtime, advisory_id: int, id: Tuple[str], noop: bool,
```